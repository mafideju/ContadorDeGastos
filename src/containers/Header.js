import React from 'react';
import { NavLink } from 'react-router-dom';

const Header = () => (
  <header>
    <h1>Carteira Virtual</h1>
    <NavLink exact to="/" activeClassName="orange">
      Home
    </NavLink>
    <NavLink to="/create" activeClassName="orange">
      Criar
    </NavLink>
    <NavLink to="/help" activeClassName="orange">
      Ajuda
    </NavLink>
  </header>
);

export default Header;
