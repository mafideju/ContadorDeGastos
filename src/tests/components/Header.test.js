import React from 'react';
import { shallow } from 'enzyme';
import Header from '../../containers/Header';

test('HEADER COMPONENT TESTING', () => {
  const wrapper = shallow(<Header />);
  expect(wrapper).toMatchSnapshot();
  // expect(wrapper.find('h1').length).toBe(1);
  // const renderer = new ReactShallowRenderer();
  // renderer.render(<Header />);
  // expect(renderer.getRenderOutput()).toMatchSnapshot();
});
