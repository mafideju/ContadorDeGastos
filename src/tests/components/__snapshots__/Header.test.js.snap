// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`HEADER COMPONENT TESTING 1`] = `
<header>
  <h1>
    Carteira Virtual
  </h1>
  <NavLink
    activeClassName="orange"
    aria-current="page"
    exact={true}
    to="/"
  >
    Home
  </NavLink>
  <NavLink
    activeClassName="orange"
    aria-current="page"
    to="/create"
  >
    Criar
  </NavLink>
  <NavLink
    activeClassName="orange"
    aria-current="page"
    to="/help"
  >
    Ajuda
  </NavLink>
  <button
    onClick={[Function]}
  >
    Sair
  </button>
</header>
`;
